SPECS/at/at.spec

```
@@ -124,8 +125,8 @@ mkdir -p %{buildroot}%{_sysconfdir}/sysconfig
 install -m 644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/atd
 
 # install systemd initscript
-mkdir -p %{buildroot}/%{_unitdir}/
-install -m 644 %{SOURCE5} %{buildroot}/%{_unitdir}/atd.service
+mkdir -p %{buildroot}/lib/systemd/system/
+install -m 644 %{SOURCE5} %{buildroot}/lib/systemd/system/atd.service
```

nit - generally try to use the %{_libdir} macro instead of /lib

```suggestion
install -m 644 %{SOURCE5} %{buildroot}/%{_libdir}/systemd/system/atd.service
```